jupyter_server/extension/application.py

```diff
@@ -311,7 +311,7 @@ def initialize_server(cls, argv=[], load_other_extensions=True, **kwargs):
         config = Config({
             "ServerApp": {
                 "jpserver_extensions": {cls.extension_name: True},
-                "open_browser": True,
+                "open_browser": cls.open_browser,
```

Comment:
I think we need to add an `open_browser` trait to ExtensionApp. Right now, this trait does not exist, so running the extension from CLI *should* fail.

Reply:
Yep, I have tested it adding the trait on my extension and it was fine, but have not thought to invoke via CLI. ExtensionApp now has a new open_browser trait.